src/one_dragon/base/operation/application/application_group_manager.py

```
@@ -97,8 +96,76 @@ def _init_one_dragon_group_config(self, instance_idx: int) -> ApplicationGroupCo
+        if daily_signin_item is not None:
+            daily_signin_item.enabled = True
+
+        # 2. 写入 daily_signin 的配置
+        try:
+            from one_dragon.base.operation.application.application_config import (
+                ApplicationConfig,
+            )
+            daily_signin_config = ApplicationConfig('daily_signin', instance_idx, config.group_id)
+            daily_signin_config.update('selected_sign', enabled_old_app_id)
+            log.info(f"已自动将每日签到的商店配置为: {enabled_old_app_id}")
+        except Exception as e:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)